pynautobot/core/api.py

```diff
@@ -74,6 +76,7 @@ def __init__(
         url,
         token=None,
         threading=False,
+        max_workers:int=4,
```

Comment:
I think given that we have type hints still in the docstring and no other type hints in the function signature, maybe we should drop these?

Reply:
fixed